lib/zip_url_to_postgis.py

```diff
@@ -31,7 +31,7 @@ def url_to_postgis(url, table_name):
         geometryType = 'Multipolygon',
         layerCreationOptions = ['precision=NO'],
         format='PostgreSQL',
-        srcSRS='EPSG:2263', 
+        srcSRS='EPSG:4326', 
```

Comment:
it should be 2263

Reply:
`doitt_zipcodeboundaries` cannot spatially join with `facilities` successfully when it's 2263

Reply:
ok